assembly-sdk-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -70,4 +70,6 @@ git.server.uri.prefix=git
 
 machine.docker.machine_extra_hosts=NULL
 
-machine.docker.che_api.endpoint=http://che-host:8080/che/api
\ No newline at end of file
+machine.docker.che_api.endpoint=http://che-host:8080/che/api
+docker.connection.tcp.connection_timeout_ms=600000
+docker.connection.tcp.read_timeout_ms=600000
```

Comment:
We have troubles with docker execs that don't return output for long time. Increasing connection timeout and read timeout is workaround that decreases suffering a bit. Good solution require big changes in docker exec logging on our side because docker doesn't support logging in execs.